django_graph_api/graphql/types.py

```diff
@@ -208,15 +208,22 @@ class Id(String):
 class Boolean(Scalar):
     @classmethod
     def coerce_result(cls, value):
+        if isinstance(value, six.string_types):
+            if value in ('False', 'false', 'f', 'F', '0'):
```

Comment:
What is the origin of these values?

I'm generally not super into the idea of coercing string values to booleans, and the spec doesn't require it, but it's also not explicitly forbidden by the spec.

I'd like to hear more about the reasoning though because I feel like you keep convincing me about things ;-)

That being said, if we do end up keeping these, what would you think of having a matching set of conversions for `true` values, and disallowing coercion of strings that don't match one of the sets? (Otherwise I'm concerned we'll end up in a situation of having 'no' get coerced to `true`, or similar.)